ksp_plugin/interface.hpp

```diff
@@ -53,6 +54,11 @@ R3Element<double> ToR3Element(XYZ const& xyz);
 WXYZ ToWXYZ(Quaternion const& quaternion);
 XYZ ToXYZ(R3Element<double> const& r3_element);
 
+not_null<std::unique_ptr<LineAndIterator>> RenderApsides(
```

Comment:
We should not have random stuff in interface.hpp.  This function should be in the unnamed namespace of interface.cpp.